package.json

```diff
@@ -1060,7 +1060,7 @@
                     "group": "Jupyter2"
                 },
                 {
-                    "when": "resourceLangId == jupyter && isWorkspaceTrusted",
+                    "when": "resourceExtname == .ipynb && isWorkspaceTrusted",
```

Comment:
This was the bug specifically, but updated the other location that I saw as well. Can't remember exactly when it changed, but this used to be valid, but now the resourceLangId is "json" so checking the extname instead.